storage/prometheus/histograms_test.go

```diff
@@ -0,0 +1,53 @@
+package prometheus
+
+import (
+	"testing"
+
+	"github.com/prometheus/prometheus/model/histogram"
+	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/tsdb/tsdbutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHistogramStatsDecoding(t *testing.T) {
```

Comment:
nit: 
```suggestion
func TestHistogramStatsIterator_Decode(t *testing.T) {
```

This makes "jump to test/implementation" work better.